asset_dashboard/views.py

```diff
@@ -22,3 +25,38 @@ def page_not_found(request, exception, template_name='asset_dashboard/404.html')
 
 def server_error(request, template_name='asset_dashboard/500.html'):
     return render(request, template_name, status=500)
+
+
+class ProjectListView(ListView):
```

Comment:
After reading [this thread](https://forum.djangoproject.com/t/understanding-how-to-combine-formview-and-listview/1879) and [the linked docs](https://docs.djangoproject.com/en/3.0/topics/class-based-views/mixins/), I'm curious whether this is the best strategy (vs. using a `CreateView` or `FormView` with a [`MultipleObjectMixin`](https://ccbv.co.uk/projects/Django/3.0/django.views.generic.list/MultipleObjectMixin/) to populate the project list). But let's put a pin in that. I think it will be easier to assess as you add more functionality.

Reply:
Yeah I wasn't sure the best approach here. Thanks for sharing those links — I will read and consider when iterating on this piece of the code.